test/widget/ui/bottom_control_navigation_bar_test.dart

```diff
@@ -33,4 +37,192 @@ void main() {
         .selectTool(brushToolData)
         .then((_) => _.checkActiveToolIconAndLabel(brushToolData));
   });
+
+  testWidgets(
+      'test if stroke width tool option is invisible when starting app',
```

Comment:
There is lots of potential for code deduplication in here that can be added to bottom_nav_bar_interactions.dart